package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "asyncios",
+  "version": "0.0.0",
+  "description": "A wrapper for promise-based library Axios into a crocks Async",
+  "main": "index.js",
```

Comment:
Do we also want to provide an esmodule version of this so that things like webpack can opt for the esmodule instead of doing the extra work to convert commonjs?

Reply:
I don't think so. I think commonjs for all the things is fine.